cpp/ql/src/Best Practices/Magic Constants/JapaneseEraDate.ql

```diff
@@ -4,7 +4,7 @@
  * @kind problem
  * @problem.severity warning
  * @id cpp/japanese-era/exact-era-date
- * @precision medium
+ * @precision low
```

Comment:
This query was mentioned in the 1.23 change note, so we'll also need to mention in the 1.24 change notes that it's been removed from the LGTM suite.

Reply:
Done.